cibuildwheel/macos.py

```diff
@@ -94,32 +97,29 @@ def call(args, env=None, cwd=None, shell=False):
         call(['pip', 'install', 'wheel'], env=env)
         call(['pip', 'install', 'delocate'], env=env)
```

Comment:
What do we do with this, then? It's (in principle) not always needed.

Reply:
I'd keep it for simplicity, 99% of users will use this.